### PR TITLE
Add VitalList.xyz

### PR DESCRIPTION
**Pull request by BrydenIsNotSmart**

### CODE REVIEW

**Review by anshtyagi0 — DISMISSED** [review]

File: data/lists/vitallist.xyz.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        "custom-bot-donate-link",
+        "custom-bot-github-link",
+        "custom-bot-invite-link",
+        "custom-bot-support-link",
+        "custom-bot-website-link",
+        "has-voting",
+        "votes-sent-to-webhooks",
+        "has-categories-or-tags",
+        "has-mobile-support",
+        "has-search",
```
Comment (MattIPv4) [inline data/lists/vitallist.xyz.json:42]: Where is the search on your site?

File: data/lists/vitallist.xyz.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    "view_bot": "https://vitallist.xyz/bots/:id",
+    "bot_widget": "https://vitallist.xyz/bots/:id/widget",
+    "content": null,
+    "owners": "BrydenIsNotSmart#0001 (529815278456930314), Kyasa#9048 (700609775838298113)",
+    "discord": "https://vitallist.xyz/discord",
+    "features": [
+        "bot-resubmission",
+        "bot-reviews",
+        "html-long-description",
+        "iframe-long-description",
```
Comment (MattIPv4) [inline data/lists/vitallist.xyz.json:31]: Your list doesn't support iframe descriptions? You allow Markdown and a very limited subset of HTML it seems

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

File: data/lists/vitallist.xyz.json (hunk 2 above)
Comment (BrydenIsNotSmart) [reply data/lists/vitallist.xyz.json:31]: We do have iframe descriptions. 
image

File: data/lists/vitallist.xyz.json (hunk 1 above)
Comment (BrydenIsNotSmart) [reply data/lists/vitallist.xyz.json:42]: Sorry, search was supposed to be added but is being delaying. I'll remove it.

File: data/lists/vitallist.xyz.json (hunk 2 above)
Comment (MattIPv4) [reply data/lists/vitallist.xyz.json:31]: Ah cool, I couldn't get it work render in the add bot preview, but looks good 👍

**Review by MattIPv4 — APPROVED** [review]

File: data/lists/vitallist.xyz.json (hunk 2 above)
Comment (BrydenIsNotSmart) [reply data/lists/vitallist.xyz.json:31]: I think we updated the bot descriptions for the bot page but not the previews, I will look into that.

**Pull request merged**